.github/workflows/discord-issue-closed.yml

```
@@ -30,12 +29,11 @@ jobs:
           AUTHOR_HTML_URL: ${{ github.event.sender.html_url }}
           TITLE: 'Closed #${{ github.event.issue.number }} ${{ github.event.issue.title }}'
           HTML_URL: ${{ github.event.issue.html_url }}
-          DESCRIPTION: ${{ github.event.issue.body }}
           REPO_FULL_NAME: ${{ github.event.repository.full_name }}
         run: |
           # This provides the env variable used by the next step
           # We have to do it like this to prevent any of the values screwing up the json
-          echo "DISCORD_EMBEDS=$(jq -nc --arg author_name "$AUTHOR" --arg author_icon_url "$AUTHOR_ICON_URL" --arg author_html_url "$AUTHOR_HTML_URL" --arg title "$TITLE" --arg html_url "$HTML_URL" --arg description "$DESCRIPTION" --arg repo_full_name "$REPO_FULL_NAME" "$TEMPLATE")" >> $GITHUB_ENV
+          echo "DISCORD_EMBEDS=$(jq -nc --arg author_name "$AUTHOR" --arg author_icon_url "$AUTHOR_ICON_URL" --arg author_html_url "$AUTHOR_HTML_URL" --arg title "$TITLE" --arg html_url "$HTML_URL" --arg repo_full_name "$REPO_FULL_NAME" "$TEMPLATE")" >> $GITHUB_ENV
```

Comment on a line

---

```suggestion
          echo "Hello"
```
Honestly, I just think a greeting in the console here would be way nicer than anything else. Maybe we could add it separately?